tests/check_framework/test_model_checks.py

```diff
@@ -395,6 +395,15 @@ class Model(models.Model):
             ),
         ])
 
+    def test_inherited__pk(self):
```

Comment:
Why do you need double underline? Is it a typo?

Reply:
Yes, it was a typo. I just fixed it.